api/reference/automation_requests.adoc

```diff
@@ -10,6 +10,7 @@ Automation requests are made available via the following entrypoint.
 ----
```

Comment:
Unrelated to your changes, but perhaps "via the following entrypoint." can be updated to "via the following entrypoint:" above